app/lib/dttp/client.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Dttp
```

Comment:
Again, moved this code from services to  `app/lib/dttp` for the same reasons as the access token code above.